project-templates/import-records/format-records-config.js

```diff
@@ -0,0 +1,290 @@
+var normalize = require('medic-bulk-utils').normalize,
+    csv = require('fast-csv'),
+    uuid = require('uuid'),
+    path = require('path');
+
+var relatedData,
+    relatedDataConfig = {
+      path: ['dist', 'people.csv'],
+      onRow: function(obj) {
+        // map a lookup key to a uuid
+        relatedData[obj.external_id] = obj.uuid;
+      }
+    };
+
+var config = {
+  columns: {
+    uuid: uuid,
+    reported_date: {
+      use: 'p_visit_date',
+      format: [normalize.date,'MM/DD/YYYY','x'] //unix ms timestamp
+    },
+    reported_date_source: 'p_visit_date',
+    person_id: 'person_id',
+    contact: function() {
+      var key = this.person_id;
+      if (typeof relatedData[key] !== 'undefined') {
+        return relatedData[key];
+      } else {
+        console.error(this);
+        throw new Error('Concession ID not found: ' + key);
```

Comment:
That's just a detail of this specific project... everyone names their places differently.